prow/cluster/jobs/istio/istio/istio.istio.master.yaml

```diff
@@ -270,6 +270,22 @@ presubmits:
           - prow/e2e-simpleTests.sh
         nodeSelector:
           testing: test-pool
+    - name: e2e-simpleTestsMinProfile-master
+      <<: *job_template
+      optional: true
```

Comment:
just a question, can you tell me the difference between "always_run" and "optional" keyword?

Reply:
Optional means required test or not.